app/api/V1/views/auth_endpoints.py

```diff
@@ -0,0 +1,139 @@
+from flask import Flask, request, jsonify, Blueprint, json, make_response
+from flask_restplus import Resource, reqparse, Api, Namespace, fields
+from ..models.user_model import User
+
+
+api = Namespace('Register Endpoint', description='A collection of register endpoints for the user model')
+ns = Namespace('Users Endpoints', description='Users endpoints to fetch all users and delete them')
```

Comment:
what does `ns` mean?

Reply:
Its a the same as api, just that I'm initializing the namespace for the user endpoints with ns